src/pages/execution/Execution.tsx

```diff
@@ -191,6 +195,10 @@ export default function Execution() {
             dag,
             selectedTask,
             setSelectedTask,
+            severity,
+            setSeverity,
+            alerts,
```

Comment:
Is there any need to bubble up the alerts themselves up to `<Execution/>`? Unless there is need for it, we should keep the state as localized as possible. E.g. can we make do with just `severity/setSeverity` to render the bubble (use undefined for none)

Reply:
In `tabLoader`, adding `cache: true` to the TabData entry for `WorkflowSummary` should cause that tab to be pre-rendered (allowing the dot to be calculated on load).